server/aws-lsp-codewhisperer/src/client/streamingClient/codewhispererStreamingClient.ts

```diff
@@ -0,0 +1,21 @@
+import { CodeWhispererStreaming } from '@amzn/codewhisperer-streaming'
+import { ConfiguredRetryStrategy } from '@aws-sdk/util-retry'
+const codeWhispererRegion = 'us-east-1'
+
+const codeWhispererEndpoint = 'https://codewhisperer.us-east-1.amazonaws.com/'
+
+export class StreamingClient {
+    public async getStreamingClient(credentialsProvider: any) {
+        return await createStreamingClient(credentialsProvider)
+    }
+}
+export async function createStreamingClient(credentialsProvider: any): Promise<CodeWhispererStreaming> {
+    const creds = credentialsProvider.getCredentials('bearer')
+    const streamingClient = new CodeWhispererStreaming({
+        region: codeWhispererRegion,
+        endpoint: codeWhispererEndpoint,
+        token: { token: creds.token },
+        retryStrategy: new ConfiguredRetryStrategy(0, (attempt: number) => 500 + attempt ** 10),
```

Comment:
Previously we set the `customUserAgent` field in the streaming client constructor for business intelligence 
purposes (https://github.com/aws/language-servers/blob/7106de3a241197e948da1312020c46c317067eff/server/aws-lsp-codewhisperer/src/language-server/chat/chatSessionService.ts#L48) 

Do you know if there is a similar expectation for transform as well?